src/elements/content-sidebar/activity-feed/comment/Comment.js

```diff
@@ -130,13 +143,7 @@ class Comment extends React.Component<Props, State> {
                     <Avatar className="bcs-comment-avatar" getAvatarUrl={getAvatarUrl} user={createdByUser} />
                     <div className="bcs-comment-content">
                         <div className="bcs-comment-headline">
-                            <UserLink
-                                className="bcs-comment-user-name"
-                                data-resin-target={ACTIVITY_TARGETS.PROFILE}
-                                id={createdByUser.id}
-                                name={createdByUser.name}
-                                getUserProfileUrl={getUserProfileUrl}
-                            />
+                            {userHeadlineRenderer ? userHeadlineRenderer(userLink) : userLink}
```

Comment:
I find this is very confusing. We're passing the `userLink` component instance to a renderer that returns a formatted message? Can this be simplified?

Reply:
Yah, this change has some stuff I ported over, so I'm going thru it now to see if there's a simpler structure